<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>IndexDialogUI</class>
 <widget class="QDialog" name="IndexDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form3</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Index Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mName" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="mMultiCheck" >
     <property name="text" >
      <string>Index Stores Multiple Records:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mCacheCheck" >
     <property name="text" >
      <string>Use Cache</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel7" >
       <property name="text" >
        <string>Hash Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox2" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="mFieldCombo" />
       </item>
       <item>
        <widget class="QPushButton" name="mAddField" >
         <property name="text" >
          <string>Add Field</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="mFieldList" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>41</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOK" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>mCancel</sender>
   <signal>clicked()</signal>
   <receiver>IndexDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>421</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel" >
     <x>422</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOK</sender>
   <signal>clicked()</signal>
   <receiver>IndexDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>299</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
